requirements.txt

```diff
@@ -2,3 +2,5 @@ simplejson
 pyserial
 pyyaml
 coveralls
+lxml
+click
```

Comment:
click is related to #356 
Unfortunately I didn't used to separated branches BUT @nhomar knows how to split these commits on merge, and he told me it was fine for such a small change :)